index.html

```diff
@@ -793,15 +786,13 @@ <h2><dfn>readable</dfn> attribute</h2>
         </ol>
 
       <li>
-        [=ReadableStream/Set up=] |stream| with
-        <a href="https://streams.spec.whatwg.org/#readablestream-set-up-pullalgorithm">pullAlgorithm</a>
+        [=ReadableStream/Set up with byte reading support=] |stream| with
+        <i><a href="https://streams.spec.whatwg.org/#readablestream-set-up-with-byte-reading-support-pullalgorithm">pullAlgorithm</a></i>
```

Comment:
Can you export these symbols from the Streams specification so hard links aren't necessary?

Reply:
They are exported; the problem is that ReSpec doesn't seem to support `for=""` values with slashes in them. /cc @marcoscaceres

Reply:
We do support it: https://respec.org/xref/?term=pullalgorithm ... but seems there is a bug.  

I'd suggest just putting in [=ReadableStream/set up/pullAlgorithm=]. I'll investigate why it's not linking.

Reply:
Filed. https://github.com/w3c/respec/issues/3595 ... hope to have a fix soon.

Reply:
Ok, should be be fixed in respec@26.8.10 😅🤞